Refactored the creation of flowcell test and its implementation

ke4 opened the pull request:
This implementation is without the dimension parameter

Commit:
Refactored the creation of flowcell test and its implementation

This implementation is without the dimension parameter

maxigit commented:
Hi Karoly.

Seems ok. However, I had a few comments on github (which mainly  
'naming convention' issue).
You should be able to see them on your commit on github.

Kind Regards
Maxime Bourget

On 30 Aug 2012, at 10:17, Károly Erdős wrote:

> This implementation is without the dimension parameter
> 
> You can merge this Pull Request by running:
> 
>   git pull https://github.com/ke4/lims-core  
> 34662429_add_createflowcell_action
> Or view, comment on, or merge it at:
> 
>   https://github.com/sanger/lims-core/pull/37
> 
> Commit Summary
> 
> Refactored the creation of flowcell test and its implementation
> File Changes
> 
> A lib/lims-core/actions/create_flowcell.rb (33)
> M lib/lims-core/laboratory/sample.rb (1)
> A spec/actions/create_flowcell_spec.rb (86)
> Patch Links
> 
> https://github.com/sanger/lims-core/pull/37.patch
> https://github.com/sanger/lims-core/pull/37.diff
> —
> Reply to this email directly or view it on GitHub.

## 

 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.

Commit:
Need the 'lims-core/resource' require

Commit:
Added the suggested changes

In spec/actions/create_flowcell_spec.rb:
- Renamed the aliquots variable to expected_aliquots
- Renamed lane_name to lane_id

In lib/lims-core/actions/create_flowcell.rb:
- Changed the comments to '@attribute [Hash<Fixnum,Array<Hash>]  ...'

Commit:
Renamed already used var name aliquots to expected_aliquots

Commit:
Changed line_id to integer in create flowcell class

maxigit commented:
Already merged
